php/Terminus/Models/Environment.php

```diff
@@ -775,21 +763,4 @@ private function getSettings($setting = null) {
 
   }
 
-  /**
-   * Changes the environment's settings
-   *
-   * @param array $settings Key/value pairs to set in the environment settings
-   * @return bool
-   */
-  private function updateSetting(array $settings = []) {
```

Comment:
Since we're moving away from it and the core does not use it, I feel confident that we can remove this function. Thanks for being so thorough!